features/step_definitions/dc.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require 'active_support/core_ext/hash/slice.rb'
-require 'active_support/core_ext/hash/transform_values.rb'
+#require 'active_support/core_ext/hash/transform_values.rb'
```

Comment:
could we just remove the line instead of commenting it out?

Reply:
@rbbratta sure, done.